notebooks/COS/ViewData/ViewData.ipynb

```diff
@@ -973,7 +1021,7 @@
     "    ax1.hist(logDQ , bins = bins_, color = 'cmy'[i], alpha = 1 - 0.25*(i) , label = segment, hatch = '|-/'[i]) # Add hatching to distinguish overlapping colors\n",
     "    ax1.legend(fontsize = 20)\n",
     "    ax1.set_yscale('log')\n",
-    "    \n",
+    "np.warnings.filterwarnings('default', category=RuntimeWarning) #suppress a divide by zero warning\n",
```

Comment:
This is the opposite of filtering out to ignore the warning - this brings the warning back in to prevent accidentally hiding important warnings later.